tests/unittests/OnnxExporterTest.cpp

```diff
@@ -483,6 +483,12 @@ TEST(exporter, onnxModels) {
       llvm::outs() << "Ignore LSTM model file: " << name << "\n";
       continue;
     }
+    if (name.find("mlpCustomOp.onnxtxt") != std::string::npos) {
```

Comment:
mlpCustomOp.onnxtxt needs to be checked in.

Reply:
Thanks. Added.